icu4c/source/test/intltest/localebuildertest.cpp

```diff
@@ -55,10 +55,17 @@ void LocaleBuilderTest::runIndexedTest( int32_t index, UBool exec, const char* &
 
 void LocaleBuilderTest::Verify(LocaleBuilder& bld, const char* expected, const char* msg) {
     UErrorCode status = U_ZERO_ERROR;
```

Comment:
FYI for the future: in tests, you can and should use IcuTestErrorCode from testlog.h.